### DISCUSSION TITLE
FR: Swtich to tabs, eventually.

### DISCUSSION THREAD

**Discussion by MattWilcox** [craftcms/cms#14575] [Ideas]
I know it's an age old debate. I know it's minor. I know it's not on the cards for 5 or any edition of.

However, can I toss out there for 6 (or whenever we switch to Laravel), we actually swap to tabs for code now that it's fairly well known to be the more accessible method.

(And for those that haven't caught up on the last few years, here's a more recent article on _why_ tabs are more accessible - https://alexandersandberg.com/articles/default-to-tabs-instead-of-spaces-for-an-accessible-first-environment/ ).

Thanks :)

**Comment by brandonkelly** [comment]
That has been on my radar, and I agree that we probably should. We are monitoring what happens with PSR standards, but may end up switching in Craft 6, even if nothing moves there.
